frontend/src/components/PromptSpeech.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import Speech from "react-speech";
+
+const PromptSpeech = (props) => {
+  const speechStyle = {
+    container: {},
+    text: {},
+    buttons: {},
+    play: {
+      hover: {
+        boxShadow: "0 0 15px 0 rgba(108,92,231,.69)",
+        backgroundColor: "GhostWhite",
+      },
+      button: {
+        width: "100",
+        height: "100",
+        cursor: "pointer",
+        pointerEvents: "none",
+        outline: "none",
+        backgroundColor: "#a29bfe",
+        border: "solid 1px #a29bfe",
+        borderRadius: 6,
+      },
+    },
+    pause: {
+      hover: {},
+      button: {},
+    },
+    stop: {
+      hover: {},
+      button: {},
+    },
+    resume: {
+      hover: {},
+      button: {},
+    },
+  };
+
+  return (
+    <Speech
+      styles={speechStyle}
```

Comment:
Ah so actually we want to be consistent for the styles. It's prob better to do use styled component instead of having it to be styles={speechStyle}. A way to do it is to create a StyledSpeech before promptSpeech component. So something like:
```
const StyledSpeech = styled(Speech)`
  :hover {
    boxShadow: "0 0 15px 0 rgba(108,92,231,.69)",
  }
`;
```
Then instead of using <Speech> u can just use <StyledSpeech>. Then you don't need to set styles here